pisound-btn/debian/DEBIAN/control

```diff
@@ -1,10 +1,9 @@
 Package: pisound-btn
-Version: 1.14-1
+Version: 1.15-1
 Section: sound
 Priority: optional
 Architecture: armhf
-Pre-Depends: hostapd (>= 2:2.6), dnsmasq (>= 2.72)
-Depends: libc6 (>= 2.19), libgcc1 (>= 1:4.9.2-10), python3 (>= 3.5), python3-urwid (>= 1.0), git (>= 1:2.1.4), libglib2.0-bin (>= 2.33.12)
+Depends: libc6 (>= 2.19), libgcc1 (>= 1:4.9.2-10), python3 (>= 3.5), python3-urwid (>= 1.0), git (>= 1:2.1.4), libglib2.0-bin (>= 2.33.12), patchbox-wifi, patchbox-cli
```

Comment:
patchbox-cli dependency should not be necessary, and it's undesired for pisound-btn to depend on it - some users may just want pisound-btn installed, without much dependencies of patchbox-os. (wifi depency is fine, as it's minimal)

Also I think it's a good idea to always set the >= for the version, I remember having some difficulties caused by it when releasing upgrades, but I can't recall the details anymore...

Reply:
Yup. Will fix. Note: I do have "Allow edits by maintainers" checked...